### DISCUSSION TITLE
How to change folder colors?

### DISCUSSION THREAD

**Discussion by blackxored** [SylEleuth/gruvbox-plus-icon-pack#86] [Q&A]
I love this theme, in the screenshot on the readme they are a bunch of colors variants for folders, I wonder how do I change them?

**Comment by SylEleuth** [comment]
I will be working on the script (and new folders icons) after next patch. For now You would have to do it manually. You can look at https://github.com/SylEleuth/gruvbox-plus-icon-pack/discussions/89 and https://github.com/SylEleuth/gruvbox-plus-icon-pack/discussions/90 for some tips.
